pkg/sink/sink.go

```diff
@@ -36,10 +39,6 @@ const (
 	KindStdout              Kind = "Stdout"
 )
 
-const (
```

Comment:
can probably also get rid of the `type Error string` `func (e Error) Error() string { return string(e) }` situation